### DISCUSSION TITLE
Question about quickstart setup step 4

### DISCUSSION THREAD

**Discussion by zaynpatel** [janhq/nitro#235] [Q&A]
I'm currently running the following command, copied from the documentation with the exception of a new localhost address:
```
url http://localhost:3000/inferences/llamacpp/loadmodel \
-H 'Content-Type: application/json' \
-d '{
    "llama_model_path": "/model/model/llama-2-7b-model.gguf",
    "ctx_len": 512,
    "ngl": 100
}'
```

I'm getting a 404 error which references that the `inferences/llamacpp/loadmodel` is not an available route. 
```
{"url":"/inferences/llamacpp/loadmodel","statusCode":404,"statusMessage":"Cannot find any route matching /inferences/llamacpp/loadmodel.","message":"Cannot find any route matching /inferences/llamacpp/loadmodel."}% 
```

I'm curious about how to proceed and wonder how I can test what other load model links might be correct?

**Comment by vassioc** [comment]
I have the same issue, only output is different

<html>

<head>
    <title>404 Not Found</title>
</head>

<body bgcolor="white" text="black">
    <center>
        <h1>404 Not Found</h1>
    </center>
    <hr>
    <center>drogon/1.9.2</center>
</body>

</html>

**Comment by louis-jan** [comment]
The URL has been changed; now, Cortex will serve the chat/completions endpoint at http://localhost:3000/inferences/server/loadmodel.
